src/main/markdown/steering.md

```diff
@@ -11,7 +11,8 @@ The steering committee consists of the following members (no particular order):
 
 * Ray Cromwell, [Google](https://www.google.com/about/)
 * Artur Signell, [Vaadin](http://www.vaadin.com)
-* Colin Alworth, [Sencha](http://www.sencha.com)
+* Justin, Hickman, [Sencha](http://www.sencha.com)
+* Colin Alworth, [RedCurrent](http://www.redcurrent.com)
```

Comment:
If the company is not really involved we don't, but final say should be up to @niloc132